documentation/cli-usage.md

```diff
@@ -0,0 +1,46 @@
+<small>*[documentation](/README.md#documentation) / kyverno-cli*</small>
+
+
+# [kyverno CLI](https://github.com/nirmata/kyverno/releases) - kubectl plugin to deal with kyverno policies
+Used to validate policies and apply policies on resources. 
+
+##Installation
+You can get the installation package of the cli for your os in the releases page [here](https://github.com/nirmata/kyverno/releases).
+
+## Commands
+
+#### Version
+Prints the version of kyverno used by the CLI.
+
+Example: 
+```
+kubectl kyverno version
+```
+
+
+#### Validate
+Validates a policy, can validate multiple policy resource description files or even an entire folder containing policy resource description 
+files. Currently supports files with resource description in yaml.
+
+Example:
+```
+kubectl kyverno validate /path/to/policy1.yaml /path/to/policy2.yaml /path/to/folderFullOfPolicies
+```
+
+#### Apply
+Applies policies on resources, supports applying multiple policies on multiple resources in a single command.
+Also supports applying the given policies to an entire cluster. Will return output to stdout. You may want to redirect
```

Comment:
Change `output` to `results` i.e. Will return results to stdout.

Remove `You may want to redirect....`